### ISSUE TITLE
isKeyDown and isKeyUp always True

### ISSUE THREAD

**Issue by barbafh3** [Anut-py/h-raylib#3]
I used the window example as base and started playing around with inputs, just printing text to see if they work. isKeyPressed and isKeyReleased are working fine, but isKeyDown and isKeyUp just return True. Heres the code i used:

```haskell
gameLoop :: IO ()
gameLoop = do
  wDown <- isKeyDown key'w
  wUp <- isKeyUp key'w
  if wDown then print "W down" else print "W not down"
  if wUp then print "W up" else print "W not up"

  beginDrawing

  clearBackground rayWhite
  drawText "Basic raylib window" 30 40 18 lightGray


  endDrawing

  shouldClose <- windowShouldClose
  unless shouldClose gameLoop
```
The output prints both "W down" and "W up" no matter the input

**Comment by Anut-py** [comment]
It looks like in the C code it is correctly returning `0` or `1` depending on whether the key is up or down. When this is marshalled into Haskell, `0` becomes `8271872` and `1` becomes `8271873`. I am trying to figure out why this happens with only these two functions and none of the other functions.

**Comment by Anut-py** [comment]
I fixed this by replacing `CInt` with `CBool` in the foreign function import. The fix is published in h-raylib version 4.5.0.4
